corehq/apps/app_manager/id_strings.py

```diff
@@ -274,6 +274,12 @@ def search_property_hint_locale(module, search_prop):
     return "search_property.m{module.id}.{search_prop}.hint".format(module=module, search_prop=search_prop)
 
 
+@pattern('search_property.m%d.%s.validation.message')
```

Comment:
Could you incorporate the validation index here? It's a little confusing that the model supports multiple validation messages, but translations don't and the UI doesn't. I think it's fine for the UI not to support it, but translations feel to me like they should match the model.

Reply:
good point, will do.